spec/requests/assignments_controller_spec.rb

```diff
@@ -145,5 +145,13 @@
       expect(project.reload.caseworker).to eq caseworker
       expect(project.reload.caseworker_assigned_at).to eq DateTime.now
     end
+
+    it "sends a notification to the caseworker" do
+      perform_request
+
+      expect(ActionMailer::MailDeliveryJob)
+        .to(have_been_enqueued.on_queue("default")
+      .with("CaseworkerAssignmentMailer", "new_assigned_caseworker", "deliver_now", args: [caseworker, Project.last]))
```

Comment:
Some indentation issues here

Reply:
This is looking good to me, apart from this! ☝️

Reply:
There isn't actually an indentation problem here - it's just github displaying it in a peculiar way.

Reply:
Same on my IDE. 6 spaces on the first line, then 8 on the next

<img width="968" alt="image" src="https://user-images.githubusercontent.com/47089130/208393164-51ea2cb5-c8d0-4b90-aa5a-36b25d1fa6aa.png">